api/migrations/README.md

```
@@ -1,59 +1,42 @@
 
-```sql
--- Rollback migration 001
-DROP TABLE IF EXISTS webhook_events;
-DROP INDEX IF EXISTS idx_users_email;
-DROP INDEX IF EXISTS idx_users_stripe_subscription;
-DROP INDEX IF EXISTS idx_users_tier;
-DROP INDEX IF EXISTS idx_users_created_at;
-DROP INDEX IF EXISTS idx_usage_records_user_id;
-DROP INDEX IF EXISTS idx_usage_records_model;
+To reset to a clean state (destroys all data):
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)